src/modules/auth/auth.service.test.ts

```diff
@@ -0,0 +1,45 @@
+import { ConfigService } from '@nestjs/config';
+import { PassportModule } from '@nestjs/passport';
+import { Test, TestingModule } from '@nestjs/testing';
+import Chance from 'chance';
+
+import { AuthService } from './auth.service';
+import { ApiKeyStrategy } from './strategy/api-key.strategy';
+
+describe('AuthService', () => {
+  let authService: AuthService;
+  const chance = new Chance();
```

Comment:
Can we use the `RandomValueGenerator` class instead please?

Reply:
updated

Reply:
this is updated now